.gitignore

```diff
@@ -10,3 +10,512 @@
 
```

Comment:
Remember to put a description in commit, not only "no message", for example in this case could be: "Files added to be ignored"